bot/resources/tags/free.md

```diff
@@ -0,0 +1,5 @@
+We recently moved to a new help channel system. There are always 2 available help channels waiting to be claimed in the **Python Help: Available category**. In order to claim one, simply start typing your question into one of these channels. Once your question has been posted, you have claimed this channel, and the channel will be moved down to the **Python Help: Occupied category**.
+
```

Comment:
I suggest changing the name to discord channel mention via id, that way if we ever decide to rename the channels, the content of the tag will be updated accordingly.
```suggestion
We recently moved to a new help channel system. There are always 2 available help channels waiting to be claimed in the **<#691405807388196926>**. In order to claim one, simply start typing your question into one of these channels. Once your question has been posted, you have claimed this channel, and the channel will be moved down to the **<#696958401460043776>**.
```